src/test/functional/queryServices.test/tags.integration.ts

```diff
@@ -317,60 +387,17 @@ describe('tags query tests - happy path', () => {
       itemId: '1',
     };
 
-    const GET_TAGS_FOR_SAVED_ITEM = gql`
-      query getSavedItem($userId: ID!, $itemId: ID!) {
-        _entities(representations: { id: $userId, __typename: "User" }) {
-          ... on User {
-            savedItemById(id: $itemId) {
-              url
-              tags {
-                ... on Tag {
-                  id
-                  name
-                  _createdAt
-                  _updatedAt
-                  _version
-                  _deletedAt
-                  savedItems {
-                    edges {
-                      cursor
-                      node {
-                        url
-                        item {
-                          ... on Item {
-                            givenUrl
-                          }
-                        }
-                      }
-                    }
-                    pageInfo {
-                      startCursor
-                      endCursor
-                      hasNextPage
-                      hasPreviousPage
-                    }
-                    totalCount
-                  }
-                }
-              }
-            }
-          }
-        }
-      }
-    `;
-
-    const res = await server.executeOperation({
+    const res = await server('1').executeOperation({
       query: GET_TAGS_FOR_SAVED_ITEM,
       variables,
     });
     expect(res.data?._entities[0].savedItemById.url).to.equal('http://abc');
     expect(res.data?._entities[0].savedItemById.tags[0].name).to.equal(
       'travel'
     );
-    const tagId0 = Buffer.from(
-      res.data?._entities[0].savedItemById.tags[0].id,
-      'base64'
-    ).toString();
+    const tagId0 = TagModel.decodeId(
```

Comment:
not a blocker: thinking if we should use the same resource for code and test?

Reply:
yeah, in this case I was taking a shortcut so I didn't have to update every test with the base64 strings 😬 

i'll update